src/components/titlePanel/createTitlePanel.ts

```diff
@@ -68,13 +79,30 @@ const createTitlePanel: TitlePanelFactory = createWidgetBase
 					};
 				}
 
+				titleProperties.role = 'heading';
+
+				if (ariaHeadingLevel) {
+					titleProperties['aria-level'] = String(ariaHeadingLevel);
+				}
+
+				if (open) {
+					// FIXME - id of content node
+					titleButtonProperties['aria-controls'] = 'content';
+				}
+
 				const children: DNode[] = [
-					v('div', titleProperties, [ title ])
+					v('div', titleProperties, [
+						v('div', titleButtonProperties, [ title ])
```

Comment:
I left it as a div since the w3 example doesn't use a button, and I wanted to avoid having to override button styles. But now I realize their example uses `<a>`, which gets much of the behavior that button has. I'd rather override button styles than reimplement button behavior, so I'll use `<button>` here.